tests/api/features/steps/boards_steps.py

```diff
@@ -0,0 +1,56 @@
+from logging import exception
+from behave import given, when, then, step
+from jsonschema import validate
+from main.core.rest.request_manager import RequestManager
+
+from main.trello.api.boards_manager import BoardsManager
+
+
+request_manager = RequestManager()
+boards_manager = BoardsManager()
+
+
+
+@step('verify there is at least 1 member')
```

Comment:
FYI:
There's a reusable common_step already implemented that includes this functionality,
It is labeled as: `'I receive a list with at least "{quantity:d}" "{item}"'`